### PR TITLE
Trunk 5052 : Extract methods from getVisitTypesToStop

### PR DESCRIPTION
**Pull request by suthagar23**
I have extracted these two methods from getVisitTypesToStop
1. getVisitTypeNamesFromGlobalPropertyValue
2. getVisitTypesFromVisitTypeNames

### CODE REVIEW

File: api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java [hunk 1]
```diff
@@ -88,6 +88,36 @@ public void setEncounterDAO(EncounterDAO dao) {
 		    dao.getEncounters(query, null, null, null, includeVoided), null);
 	}
 	
+
+	private void failIfDeniedToEdit(Encounter encounter) throws APIException {
+		throw new APIException("Encounter.error.privilege.required.edit", new Object[] { encounter.getEncounterType()
```
Comment (teleivo) [inline api/src/main/java/org/openmrs/api/impl/EncounterServiceImpl.java]: i think the changes in this file are not part of the issue. this comes from your other PR. please fix that.

**Comment by teleivo** [comment]
@suthagar23 are you working on this? We have a lot of PRs open and would like to merge them

**Comment by dkayiwa** [comment]
@suthagar23 can you squash these commits into one as advised at https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by teleivo** [comment]
same comments as https://github.com/openmrs/openmrs-core/pull/2129

**Pull request closed without merging**